### PR TITLE
add landing page for WeAreDevelopers conference, add event to resources

### PR DESCRIPTION
**Pull request by shughes26**
### Proposed changes
Created landing page for https://github.com/pulumi/marketing/issues/1070 and corresponding `/resources` entry

### CODE REVIEW

**Review by AnDresChoContA12 — COMMENTED** [review]
My apologies the Meta Image had the wrong date so I replaced the incorrect one
WAD-BERLIN MEET US (1200 x 628 px) (1)
Other than that I am okay with it!

File: content/resources/we-are-developers/index.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+# Name of the event, <= 60 characters
+title: WeAreDevelopers Berlin - July 17-19, 2024
```
Comment (SaraDPH) [inline content/resources/we-are-developers/index.md:3]:
```suggestion
title: WeAreDevelopers Berlin
``

Comment (SaraDPH) [reply content/resources/we-are-developers/index.md:3]: The date shouldn't be in the title.

**Comment by SaraDPH** [comment]
The link to the meet us block changed to https://info.pulumi.com/meet-us/wearedevelopers

**Review by AnDresChoContA12 — APPROVED** [review]
Thank you!

**Pull request merged**